disnake/appinfo.py

```diff
@@ -227,7 +240,12 @@ class PartialAppInfo:
     summary: :class:`str`
         If this application is a game sold on Discord,
         this field will be the summary field for the store page of its primary SKU.
-    verify_key: :class:`str`
+
+        .. deprecated:: 2.5
+
+            This field is deprecated by discord and is now always blank. Consider using :attr:`.description` instead.
+
+        verify_key: :class:`str`
```

Comment:
This line has 4 extra leading spaces

Reply:
https://github.com/DisnakeDev/disnake/pull/369/commits/547dc0aa6c57b309d9556207632573cc1107dc78